cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/replication/ReplicationTestBase.java

```diff
@@ -178,11 +178,11 @@ protected void assertCookieInterceptorPresent(ReplicatorBuilder p, String expect
         respI.setAccessible(true);
         List<HttpConnectionRequestInterceptor> reqIList = (List)reqI.get(p);
         List<HttpConnectionRequestInterceptor> respIList = (List)respI.get(p);
-        Assert.assertEquals(1, reqIList.size());
-        Assert.assertEquals(CookieInterceptor.class, reqIList.get(0).getClass());
+        Assert.assertEquals(2, reqIList.size());
```

Comment:
Since this test is relying on the `UserAgentInterceptor` being added before the `CookieInterceptor`
I think it might be worth having a comment here making that assumption clear in case anyone needs to debug this later.